sys/geom/geom_ctl.c

```diff
@@ -254,15 +259,15 @@ gctl_dump(struct gctl_req *req)
 	for (i = 0; i < req->narg; i++) {
 		ap = &req->arg[i];
 		if (!(ap->flag & GCTL_PARAM_NAMEKERNEL))
-			printf("  param:\t%d@%p", ap->nlen, ap->name);
+			printf("  param:\t???");
```

Comment:
These are both fine to print, it's not derefencing anything other than `ap` itself (it's `%p` not `%s`).

Reply:
Will fix.  This was earlier in the series and in later ones I just added casts to get back to a void * to pass to %p.